typeset/tests/typeset.test.js

```diff
@@ -60,6 +60,8 @@ afterAll(() => {
   }
 })
 
+//add another test that calls new function to check input and output also update the createMap ones below
```

Comment:
As an additional way to verify that the tests cover all of the new code, the `[Files changed]` tab in the Pull Request shows all of the uncovered lines of code